### PR TITLE
Adds ODL HA support

### PR DESCRIPTION
**Pull request by trozet**

### CODE REVIEW

File: manifests/config.pp [hunk 1]
```diff
@@ -51,4 +51,30 @@
...
+    if $ha_node_count >= 2 {
+      # Configuration Jolokia XML for HA
+      file { 'jolokia.xml':
+        ensure  => file,
+        path    => '/opt/opendaylight/deploy/jolokia.xml',
+        # Set user:group owners
+        owner   => 'odl',
+        group   => 'odl',
+        # Use a template to populate the content
+        content => template('opendaylight/jolokia.xml.erb'),
```
Comment (dfarrell07) [inline manifests/config.pp]: Since the content of this file isn't dynamic I think we don't need a template.

File: manifests/init.pp [hunk 2]
```diff
@@ -19,6 +19,12 @@
 #   Enable or disable ODL OVSDB ML2 L3 forwarding. Valid: true, false, 'yes' and 'no'.
 # [*log_levels*]
 #   Custom OpenDaylight logger verbosity configuration (TRACE, DEBUG, INFO, WARN, ERROR).
+# [*enable_ha*]
+#   Enable or disable ODL OVSDB HA Clustering. Valid: true or false. Default: false.
+# [*ha_node_ips*]
+#   Array of IPs for each node in the HA cluster.
+# [*ha_node_index*]
```
Comment (dfarrell07) [inline manifests/init.pp]: Going to need better docs for this, but I don't mind handling that in a follow-up myself (also need to add docs to README).

**Comment by trozet** [comment]
@dfarrell07 Updated with the fix for using a file instead of template.

**Comment by dfarrell07** [comment]
All CI tests are failing due to an indentation problem. Fixing myself, np.

**Comment by trozet** [comment]
Thanks!

**Pull request merged**